From Larry.Finger@lwfinger.net  Thu Nov 19 11:55:55 2009
From: Larry Finger <Larry.Finger@lwfinger.net>
Date: Mon, 16 Nov 2009 23:35:38 -0600
Subject: Staging: rtl8187se: Rename staging driver to avoid name conflict with mainline driver
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <4b02362a.0nr661Zozk1yE8rZ%Larry.Finger@lwfinger.net>


Now that active development has begun on a mainline version of
a driver for the RTL8187SE that should be called rtl8187se, there
is a conflict with the driver in staging with the same name.

To solve the conflict, rename the driver in staging to r8187se.

Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/Makefile           |    2 +-
 drivers/staging/rtl8187se/Kconfig  |    3 ++-
 drivers/staging/rtl8187se/Makefile |    4 ++--
 3 files changed, 5 insertions(+), 4 deletions(-)

--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -19,7 +19,7 @@ obj-$(CONFIG_COMEDI)		+= comedi/
 obj-$(CONFIG_ASUS_OLED)		+= asus_oled/
 obj-$(CONFIG_PANEL)		+= panel/
 obj-$(CONFIG_ALTERA_PCIE_CHDMA)	+= altpciechdma/
-obj-$(CONFIG_RTL8187SE)		+= rtl8187se/
+obj-$(CONFIG_R8187SE)		+= rtl8187se/
 obj-$(CONFIG_RTL8192SU)		+= rtl8192su/
 obj-$(CONFIG_RTL8192E)		+= rtl8192e/
 obj-$(CONFIG_INPUT_MIMIO)	+= mimio/
--- a/drivers/staging/rtl8187se/Kconfig
+++ b/drivers/staging/rtl8187se/Kconfig
@@ -1,6 +1,7 @@
-config RTL8187SE
+config R8187SE
 	tristate "RealTek RTL8187SE Wireless LAN NIC driver"
 	depends on PCI && WLAN
 	depends on WIRELESS_EXT
 	default N
 	---help---
+	  If built as a module, it will be called r8187se.ko.
--- a/drivers/staging/rtl8187se/Makefile
+++ b/drivers/staging/rtl8187se/Makefile
@@ -16,7 +16,7 @@ EXTRA_CFLAGS += -DENABLE_LPS
 
 #EXTRA_CFLAGS += -mhard-float -DCONFIG_FORCE_HARD_FLOAT=y
 
-rtl8187se-objs :=			\
+r8187se-objs :=			\
 		r8180_core.o		\
 		r8180_93cx6.o		\
 		r8180_wx.o		\
@@ -35,5 +35,5 @@ rtl8187se-objs :=			\
 		ieee80211/ieee80211_crypt_ccmp.o	\
 		ieee80211/ieee80211_crypt_wep.o
 
-obj-$(CONFIG_RTL8187SE)	+= rtl8187se.o
+obj-$(CONFIG_R8187SE)	+= r8187se.o
 
